beancount/core/account.py

```diff
@@ -195,6 +195,9 @@ def walk(root_directory: Account) -> Iterator[Tuple[str, Account, List[str], Lis
     This only yields directories that are accounts... it skips the other ones.
     For convenience, it also yields you the account's name.
 
+    Note:
+        this only work as expected on POSIX.
+
```

Comment:
Why is it so? I don't see anything POSIX specific in this function.

Reply:
`os.walk` is available on Windows too.

Do you realize that I am putting it time and effort to review your patches and that this style of replied to my comments is really annoying? I have tried to convey this several times now. I understand that English is not your native language, it is not mine either. For this reason it would be better to spend more words, not less, to explain your position. From your reply above the only conclusion I can draw is that you have no idea of what you are writing about.

Reply:
Ah, you are right. the path seprator of root by yield `account.walk` doesn't matter here.

At least we do handle it as filepath this time.